src/cargo/util/command_prelude.rs

```diff
@@ -1,6 +1,25 @@
+use std::ffi::{OsStr, OsString};
```

Comment:
Although I love this "standard" way to arrange import orders, could we follow atomic commit pattern and leave off this to a separate commit or pull request?

Reply:
This wasn't an intended change, and simply auto-refactoring of the imports. I also (usually) prefer not to mess with imports to avoid merge conflicts. Rolled back this change :+1: